SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -18,6 +18,10 @@ component UnrealServerRPCEndpointLegacy {
     bool ready = 1;
     event UnrealRPCPayload server_to_client_rpc_event;
     event UnrealPackedRPCPayload packed_server_to_client_rpc;
+}
+
+component UnrealServerToServerRPCEndpointLegacy {
```

Comment:
Should we call this legacy if we don't have something to replace it yet. Generally I'd rather give something a descriptive name rather than legacy. But I guess that is just following what we've done elsewhere.

Reply:
renamed to `UnrealServerToServerCommandEndpoint`